lib/ddtrace/contrib/rails/framework.rb

```diff
@@ -11,17 +11,6 @@
 require 'ddtrace/contrib/rails/active_support'
 require 'ddtrace/contrib/rails/utils'
 
-# Rails < 3.1
```

Comment:
I guess that way we're dropping support for Rails < 3.1. It's best-effort support at the moment, so probably something we can't avoid.